elixir-mode.el

```diff
@@ -330,6 +305,33 @@ for the Elixir programming language."
     (,(elixir-rx (group sigils))
      1 font-lock-builtin-face)
 
+    ;; Sigil patterns. Elixir has support for eight different sigil delimiters.
```

Comment:
I wasnt sure if sigils and regexes should be highlighted differently. 

Should they look the same then ?